defaults/main.yml

```diff
@@ -44,3 +44,7 @@ openwisp2_topology_update_frequency:
     day: "*"
     hour: "*"
     minute: "*/5"
+openwisp2_topology_save_snapshot_frequency:
+    day: "*/1"
+    hour: "0"
+    minute: "0"
```

Comment:
let's start this process at 23:30 by default, that way we will capture the state of the network during the last hours of the day